config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -0,0 +1,71 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: soak-tests-capz-windows-2019
+    decorate: true
+    always_run: false
+    optional: true
+    decoration_config:
+      timeout: 8h
+    path_alias: sigs.k8s.io/kubernetes
+    trigger: "(?m)soak test this( please)?"
+    rerun_command: "soak test this please"
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
+      preset-azure-windows: "true"
+      preset-capz-windows-common-main: "true"
```

Comment:
now that this a Presubmit and not a periodic you will want these labels: https://github.com/kubernetes/test-infra/blob/8090d73acd13d2f4f533b7c545b5411297e327b1/config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml#L81-L83

Sorry about the churn there